src/Sparrow.Server/Sparrow.Server.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1</TargetFrameworks>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
@ppekrol I'm pretty sure that this is fine to do. Can you verify?

Reply:
Should be fine. It is used only in Raven.Server which is a netcoreapp3.1 project.